ide-common/src/main/kotlin/org/digma/intellij/plugin/ui/service/ErrorsViewService.kt

```diff
@@ -50,9 +51,39 @@ class ErrorsViewService(project: Project) : AbstractViewService(project) {
         return "Errors" + if (model.errorsCount > 0) " (${model.count()})" else ""
     }
 
+
+
+    //todo: quick and dirty prevent race condition with refresh task ultil we have time to re-write it
```

Comment:
```suggestion
    //todo: quick and dirty prevent race condition with refresh task until we have time to re-write it
```

please modify all the **ultil** to **until**